src/locale/sk-SK.h

```diff
@@ -0,0 +1,393 @@
+// Copyright 2019 - David Conran (@crankyoldgit)
+// Locale/language file for Slovakia.
+// This file will override the default values located in `defaults.h`.
+#ifndef LOCALE_SK-SK_H_
+#define LOCALE_SK-SK_H_
```

Comment:
As the build failure says, that dash `-` should be an underscore `_`...

Reply:
Yep.
```
src/locale/sk-SK.h:4:  #ifndef header guard has wrong style, please use: LOCALE_SK_SK_H_  [build/header_guard] [5]
src/locale/sk-SK.h:3[9](https://github.com/crankyoldgit/IRremoteESP8266/actions/runs/8870139943/job/24351598938?pr=2091#step:6:10)3:  #endif line should be "#endif  // LOCALE_SK_SK_H_"  [build/header_guard] [5]
```